undefined: MustAssetString

Hello,

I can't build the frontend here the error:
```
go get -u github.com/kevinburke/go-bindata/...
go generate
go build -ldflags "-w -s" -o frontend
# github.com/xddxdd/bird-lg-go/frontend
./template.go:107:10: undefined: MustAssetString
*** Error code 2
```
 **Test environment:** 
-  FreeBSD/Arch Linux/Ubuntu
- Go1.16/1.15

Kind regards.

Hello,

Here the log..
```
[21:53] [dev1@existful]-[~/bird-lg-go]: go env
GO111MODULE="on"
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/dev1/.cache/go-build"
GOENV="/home/dev1/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="freebsd"
GOINSECURE=""
GOMODCACHE="/home/dev1/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="freebsd"
GOPATH="/home/dev1/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/freebsd_amd64"
GOVCS=""
GOVERSION="go1.16.2"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/dev/null"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build570789875=/tmp/go-build -gno-record-gcc-switches"
```

```
[21:54] [dev1@existful-[~/bird-lg-go]: make
make -C frontend all
go get -u github.com/kevinburke/go-bindata/...
go get: added github.com/kevinburke/go-bindata v3.22.0+incompatible
go generate
go build -ldflags "-w -s" -o frontend
# github.com/xddxdd/bird-lg-go/frontend
./template.go:107:10: undefined: MustAssetString
*** Error code 2

Stop.
make[1]: stopped in /usr/home/vr0/bird-lg-go/frontend
*** Error code 1

Stop.
make: stopped in /usr/home/vr0/bird-lg-go
```
Kind regards.

--- a/README.md
+++ b/README.md
@@ -42,6 +42,8 @@ go build -ldflags "-w -s" -o proxy
 cd ..
 ```
 
+- If you get `undefined: MustAssetString`, you need to uninstall an older version of go-bindata from your machine: see #11
+
 ## Frontend
 
 The frontend directory contains the code for the web frontend, where users see BGP states, do traceroutes and whois, etc. It's a replacement for "lg.py" in original bird-lg project.

